yt_dlp/extractor/pornez.py

```diff
@@ -3,7 +3,7 @@
 
 
 class PornezIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?pornez\.net/video(?P<id>[0-9]+)/'
+    _VALID_URL = r'https?://(?:www\.)?pornez\.net/(video(?P<id>[0-9]+)|watch)/'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?pornez\.net/(?:video(?P<id>[0-9]+)|watch)/'
```

Reply:
this PR could also fix #6298, the id can be alphanumeric